Sources/MessagingPush/Background Queue/QueueRunner.swift

```diff
@@ -4,9 +4,9 @@ import Foundation
 // Queue tasks for the MessagingPush module.
 // sourcery: InjectRegister = "QueueRunnerHook"
 public class MessagingPushQueueRunner: ApiSyncQueueRunner, QueueRunnerHook {
-    init(siteId: SiteId, diGraph: DICommon) {
-        super.init(siteId: siteId, jsonAdapter: diGraph.jsonAdapter, logger: diGraph.logger,
-                   httpClient: diGraph.httpClient)
+    override init(siteId: SiteId, jsonAdapter: JsonAdapter, logger: Logger, httpClient: HttpClient) {
```

Comment:
No more having to do hacks like this where you must inject a DI graph in the constructor. Now all classes can simply inject all dependencies via the constructor!